views/confirmation/review.pug

```diff
@@ -23,7 +73,7 @@ block content
           +validationMessage(errors.review.msg, key)
         .multiple-choice__item
           input(id='review' name='review' type="checkbox" value="review" checked=(value =='review') aria-describedby=(errors ? `review-error` : false))
-          label(for='review') I totally consent to this
+          label(for='review') I confirm that, as far as I know, all the information in this tax return is accurate
```

Comment:
Does this get translated? It doesn't seem to be part of en.json

Reply:
Good catch ⚾️